profile.html

```diff
@@ -20,23 +20,25 @@
 	<div class="container">
 		<div class="profile-body-container">
 			<div class="grid-item">
-				<div id="profile-box-grid">
-					<div>
-						<img id="profile-picture" src="images/geo.jpg" alt="Profile picture">
-					</div>
-					<div id="brief-profile-description">
-						<span class="bigger">Georgian</span> <br>
-						<span class="bigger">Popescu</span> <br>
-						<small>here since Mar 2018</small>
-					</div>
-						<div id="profile-description">
-						<span class="big">
-							Achievements : 3/10<br>
-							Courses : 1/3<br>
-							Time spent learning : 2 h<br>
-							Longest streak : 1 days<br>
-							Title : Newbie
-						</span>
+				<div class="grid-itemm">
```

Comment:
Please choose a better name for this class

Reply:
Sorry , I forgot to .